runelite-client/src/main/java/net/runelite/client/plugins/hunter/HunterPlugin.java

```diff
@@ -178,8 +195,44 @@ public void onGameObjectSpawned(GameObjectSpawned event)
 						notifier.notify("You've caught part of a monkey's tail.");
 					}
 				}
+				else
+				{
+					// Due to the rotation of net traps, we don't store the correct tile location unless the trap
+					// faces east. We have to look at adjacent tiles to locate the traps properly.
+					final WorldPoint objectTile = gameObject.getWorldLocation();
 
+					WorldPoint northTest = new WorldPoint(objectTile.getX(), objectTile.getY() + 1, objectTile.getPlane());
```

Comment:
Wouldn't it be easier just to check what rotation the object has and determine it based on that?

Reply:
Good point. Working on an alternative solution using the rotation.